xarray/core/concat.py

```diff
@@ -112,14 +112,7 @@ def concat(
         raise ValueError("must supply at least one object to concatenate")
 
     if dim is None:
```

Comment:
See #3288 (I changed `dim=None` to `dim`) in the signature. LGTM otherwise. Feel free to merge

Reply:
Ah great, thanks